### DISCUSSION TITLE
How i can i make camera transition tween zoom but not position?

### DISCUSSION THREAD

**Discussion by JelmerBerghs** [ramokz/phantom-camera#331] [Question / Help Wanted] [answered]
I have 2 phantomcameras.
1 main camera that follows the players.
1 camera that gets a higher priority when the player is sprinting, it has identical properties to the main camera except for the zoom.

How can i make it so it only tweens the zoom of the camera and not the position. (player stays exactly in center at all times).
![image](https://github.com/ramokz/phantom-camera/assets/22679293/04754623-51e9-49af-87bf-63edb9691bb4)
![image](https://github.com/ramokz/phantom-camera/assets/22679293/46a37847-8d68-4263-8341-58aeec8418c0)

https://github.com/ramokz/phantom-camera/assets/22679293/0e6e3f8d-49b8-45b0-bb5d-85bcee3d43a3

**Comment by JelmerBerghs** [comment]
Code:
![image](https://github.com/ramokz/phantom-camera/assets/22679293/19f30080-6176-48c0-9df6-74d7588c00f6)

**Comment by ramokz** [comment, marked as answer]
If you're only modifying the `Zoom`, couldn't you just stick to using just one `PCam` and then just manually tween the `Zoom` property using Godot's built-in [Tween](https://docs.godotengine.org/en/latest/classes/class_tween.html)?

**Reply by JelmerBerghs** [reply]
Yes indeed. Thats what i'll do.

I'm just experimenting with the addon. I thought i was maybe missing something. But that solution is all i need. 
Thanks you so much, loving the addon. Amazing work!
